src/SplitPane.vue

```diff
@@ -38,12 +41,22 @@ function dragMove(e: MouseEvent) {
       : container.value.offsetWidth
     const dp = position - startPosition
     state.split = startSplit + +((dp / totalSize) * 100).toFixed(2)
+
+    const viewElement = rightView.value.querySelector('iframe')
+    state.viewHeight = viewElement.offsetHeight
+    state.viewWidth = viewElement.offsetWidth
```

Comment:
- Avoid using DOM API `querySelector`, use template ref & `defineExpose` instead.
- Extract these 3 lines into a function.

Reply:
I tried it, but it seemed a bit cumbersome.